### DISCUSSION TITLE
Updating the Firmware - please help

### DISCUSSION THREAD

**Discussion by vaderag** [kpine/zwave-js-server-docker#24] [General]
Okay, I'm close to flinging my USB stick in the bin...

I'm following @kpine's guide here: https://github.com/kpine/zwave-js-server-docker/wiki/700-series-Controller-Firmware-Updates-(Linux)

I've tried the Windows way (see my fun and games on that over on the [HA forum](https://community.home-assistant.io/t/help-with-updating-z-wave-firmware/452468/5)

and now I'm trying the Linus way on a VM

TL;DR in both routes I seem to get stuck on Xmodem - on PC 

> Can't open Xmodem session

and on Linux 

> Give your local XMODEM receive command now

I'm really at a loss - I have never known it to be so difficult to update a firmware on ANY device

**Comment by vaderag** [comment]
I've done it... the driver must have been corrupt or something I decided to remove it entirely. Re-installed (from the same package I installed it from fwiw) and then suddenly everything was working
